tests/benchmarks/test_h2o.py

```diff
@@ -1,42 +1,50 @@
 """
 h2o-ai benchmark groupby part running on coiled.
 
-Note: Only holistic aggregations (median and groupby-apply) use a shuffle with the default split_out=1.
+Note: Only holistic aggregations (median and groupby-apply) use a shuffle with the
+default split_out=1.
 """
+import os
 
-import dask
 import dask.dataframe as dd
 import pandas as pd
 import pytest
-from packaging.version import Version
-
-from ...utils_test import run_up_to_nthreads
-
-
-@pytest.fixture(
-    scope="module",
-    params=[
-        "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_single.csv",
-        # "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_single.csv",
-        # "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_single.csv",
-        "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_parquet/*.parquet",
-        "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_parquet/*.parquet",
-        # "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_parquet/*.parquet",
-    ],
-    ids=[
-        "0.5 GB (csv)",
-        # "5 GB (csv)",
-        # "50 GB (csv)",
-        "0.5 GB (parquet)",
-        "5 GB (parquet)",
-        # "50 GB (parquet)",
-    ],
-)
-def ddf(request):
-
-    if request.param.endswith("csv"):
+
+from ..utils_test import run_up_to_nthreads
+
+DATASETS = {
+    "0.5 GB (csv)": "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_single.csv",
+    "5 GB (csv)": "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_single.csv",
+    "50 GB (csv)": "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_single.csv",
+    "0.5 GB (parquet)": "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_parquet/*.parquet",
+    "5 GB (parquet)": "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_parquet/*.parquet",
+    "50 GB (parquet)": "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_parquet/*.parquet",
+}
```

Comment:
We also have the datasets that have `id1`, `id2` and `id3` as pyarrow strings for 5GB, 50GB, and even 500GB. I can create 0.5GB if needed, the s3 uri are:

- 5GB uri: `s3://coiled-datasets/h2o-benchmark/pyarrow_strings/N_1e8_K_1e2/`
- 50GB uri: `s3://coiled-datasets/h2o-benchmark/pyarrow_strings/N_1e9_K_1e2/`
- 500GB: `s3://coiled-datasets/h2o-benchmark/pyarrow_strings/N_1e10_K_1e2/`